linera-base/src/ownership.rs

```diff
@@ -156,8 +156,17 @@ impl ChainOwnership {
     }
 
     /// Returns `true` if this is an owner or super owner.
-    pub fn verify_owner(&self, owner: &AccountOwner) -> bool {
-        self.super_owners.contains(owner) || self.owners.contains_key(owner)
+    pub fn is_owner(&self, owner: &AccountOwner) -> bool {
+        self.super_owners.contains(owner)
+            || self.owners.contains_key(owner)
+            || self.first_leader.as_ref().is_some_and(|fl| fl == owner)
+    }
+
+    /// Returns `true` if this is an owner or if `open_multi_leader_rounds`.
+    pub fn is_multi_leader_owner(&self, owner: &AccountOwner) -> bool {
+        self.open_multi_leader_rounds
```

Comment:
This will return `true` even if `owner` isn't even't in `self.owners` or `self.super_owners` collections.